st2common/st2common/openapi.yaml

```diff
@@ -0,0 +1,3781 @@
+swagger: '2.0'
+
+info:
+  version: "1.0.0"
+  title: StackStorm API
+
+paths:
+  /api/v1/:
+    get:
+      operationId: st2api.controllers.root:root_controller.index
+      description: General API info.
+      responses:
+        '200':
+          description: General API info.
+          schema:
+            type: object
+            properties:
+              version:
+                type: string
```

Comment:
We need A LOT more descriptions and examples in this file. It's going to be a long journey.